src/knock.ts

```diff
@@ -172,7 +172,10 @@ class Knock {
           throw new BadRequestException(code, message, requestID);
         }
         case 422: {
-          const { errors } = data;
+          const errors = Array.isArray(data.errors)
+            ? data.errors
+            : [data.errors];
```

Comment:
If errors is null or undefined, then won't this end up as `[null]`?

Reply:
Good call, added an extra check if errors is not defined